cases/templates/cases/notifications/notification_list.html

```diff
@@ -138,5 +138,12 @@ <h1>Notifications</h1>
     </script>
 {% endif %}
 
+<section class="nw-page-announcement nw-page-announcement--info">
+    <div class="nw-page-announcement__content">
+        You can change your notification settings
+        <a href="{% url "accounts:staff-settings" %}">here</a>.
```

Comment:
I'd have this as something like "You can [change your notification settings]" - try to avoid "here" and the like for the reasons given at e.g. https://www.gov.uk/guidance/content-design/links